src/data/data.js

```diff
@@ -46,130 +48,143 @@ export const posts = [
     id: 1,
     title: "post-pic",
     user: avatar1,
-    location: "Iran , ardabil",
+    location: "Iran , Ardabil",
     name: "nick",
     likes: 2300,
     comments: 20,
+    position: [37.505, 48.5],
   },
   {
     image: post2,
     id: 2,
     title: "post-img",
     user: avatar2,
-    location: "Iran , tehran",
+    location: "Iran , Tehran",
     name: "john",
     likes: 1000,
     comments: 120,
+    position: [35.505, 51.5],
   },
   {
-    image: post3,
+    image: post7,
     id: 3,
     title: "post-image",
     user: avatar3,
-    location: "Iran , tabriz",
+    location: "Iran , Tabriz",
     name: "mahsa",
     likes: 22300,
     comments: 810,
+    position: [37.505, 46.5],
   },
   {
     image: post4,
     id: 4,
     title: "post-sth",
     user: avatar4,
-    location: "Iran , gilan",
+    location: "Iran , Gilan",
     likes: 6300,
     name: "ali",
     comments: 310,
+    position: [36.505, 47.5],
   },
   {
-    image: post2,
+    image: post4,
     id: 5,
     title: "post-img",
     user: avatar2,
-    location: "Iran , tehran",
+    location: "Iran , Tehran",
     name: "john",
     likes: 1000,
     comments: 120,
+    position: [35.505, 51.5],
   },
   {
     image: post3,
     id: 6,
     title: "post-image",
     user: avatar3,
-    location: "Iran , tabriz",
+    location: "Iran , Tabriz",
```

Comment:
that space before comma is wrong, it should be `Iran, Tabriz` remove it for all these cases